packages/pigeon/lib/ast.dart

```diff
@@ -139,6 +140,7 @@ class AstProxyApi extends Api {
     required this.fields,
     this.superClass,
     this.interfaces = const <TypeDeclaration>{},
+    this.kotlinOptions,
```

Comment:
Since the generated code references the native class, each ProxyApi needs to provide the full class name with the package. I also think `Method.swiftFunction` is similar since it provides additional instructions for the Swift generator.